matplotlibrc.template

```diff
@@ -352,6 +352,8 @@ backend      : $TEMPLATE_BACKEND
 #date.autoformatter.second   : %H:%M:%S
 #date.autoformatter.microsecond   : %M:%S.%f
 
+#autodatelocator.spacing : tight ## if generous, add extra spacing to avoid overlap, otherwise tight
```

Comment:
Maybe `autodatelocator.generousspacing : False` True/False would be a cleaner API?